On really fast machines some of the .o files aren't being added to
libray.a because they are generated so fast that make(1) thinks the
library is up-to-date.  Marvel at the horrific hack below.

--- libray/Makefile.orig	Sat Feb 26 16:41:28 2000
+++ libray/Makefile	Sat Feb 26 16:42:36 2000
@@ -12,6 +12,8 @@
 	for i in $(STUFF); do \
 		(cd $$i && $(MAKE)); \
 	done
+	rm -f libray.a
+	ar cur libray.a `find . -name '*.o'`
 
 #
 # Have to keep clean/depend separete so
